tests/e2e/specs/in-game/game-over/rematch.spec.js

```diff
@@ -121,6 +121,7 @@ describe('Creating And Updating Ranked Matches With Rematch', () => {
 
     // Match should be incomplete with one game
     cy.request('http://localhost:1337/match').then((res) => {
+      debugger;
       expect(res.body.length).to.eq(3);
```

Comment:
@itsalaidbacklife this line is failing now in this branch. Where is `/match` defined as a route? I can't find it anywhere.